### PR TITLE
open sourcing ats-zeus component

### PR DESCRIPTION
**Pull request by dmorilha**
@sc0ttbeardsley This open sources ats-zeus. It is not hooked up into the build just yet

### CODE REVIEW

File: ats/ats-zeus.cc [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+      TS_STAT_NON_PERSISTENT, TS_STAT_SYNC_COUNT);
+}
+
+TSReturnCode TSRemapInit(TSRemapInterface *, char *, int) {
+  TSDebug(PLUGIN_TAG, "init");
+
+  TSPluginRegistrationInfo info;
+
+  info.plugin_name = const_cast< char * >(PLUGIN_TAG);
+  info.support_email = const_cast< char * >("search-l7-dev@yahoo-inc.com");
```
Comment (sc0ttbeardsley) [inline ats/ats-zeus.cc]: need to change this

File: ats/ats-zeus.cc [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+  info.vendor_name = const_cast< char * >("Yahoo Inc");
```
Comment (sc0ttbeardsley) [inline ats/ats-zeus.cc:64]: Maybe this too?

File: ats/remap.config [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2015, Yahoo Inc. All rights reserved.
+# Copyrights licensed under the New BSD License.
+# See the accompanying LICENSE file for terms.
+
+map /config2 http://www.yahoo.com/ @plugin=ats-zeus.so @pparam=/usr/share/zeus/config.so.2
```
Comment (sc0ttbeardsley) [inline ats/remap.config]: make example.com ?

**Comment by sc0ttbeardsley** [comment]
lgtm w/ comments

File: ats/ats-zeus.cc (hunk 1 above)
Comment (dmorilha) [inline ats/ats-zeus.cc]: what's the suggestion here?

File: ats/ats-zeus.cc (hunk 2 above)
Comment (dmorilha) [inline ats/ats-zeus.cc:64]: ditto

**Pull request merged**